### PR TITLE
boards: added missing arduino definitions to nRF5340 Audio DK board.

### PR DESCRIPTION
**Pull request by smaerup**
Added missing Arduino ADC, I2C and SPI definitions to nrf5340_audio_dk_nrf5340 board.

### CODE REVIEW

**Comment by larsgk** [comment]
Just a small comment for reviewers: @smaerup works in our team in Demant and I have the board he made work with the TFT shield (how we saw the bug) - now it runs smoothly, so r+ from me

**Review by larsgk — DISMISSED** [review]
The TFT shield now works with the audio DK, so I'd love to see this go in.

File: boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi [hunk 1]
```diff
@@ -49,11 +49,22 @@
 			   <16 0 &gpio1 12 0>,	/* D10 */
 			   <17 0 &gpio0 9 0>,	/* D11 */
 			   <18 0 &gpio0 10 0>,	/* D12 */
-			   <19 0 &gpio0 8 0>,	/* D13 */
+			   <19 0 &gpio0 28 0>,	/* D13 */
```
Comment (koffes) [inline boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi:52]: This should be left the way it was, which is correct according to the Audio DK (PCA10121) Schematics. Why is this moved?

**Review by koffes — CHANGES_REQUESTED** [review]
Thanks for the PR. This looks good. Just one change requested.

File: boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi (hunk 1 above)
Comment (smaerup) [reply boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi:52]: My bad, I looked at the silkscreen, and apparently my eyes are not what they sued to be :-)

**Review by rick1082 — APPROVED** [review]

**Review by larsgk — APPROVED** [review]

**Comment by koffes** [comment]
Please remember to re-request review :) I wondered why I did not get another notification about this. Approved.

**Review by koffes — APPROVED** [review]

**Review by larsgk — DISMISSED** [review]

**Review by rick1082 — APPROVED** [review]

**Comment by smaerup** [comment]
Ran in to a few strange twister issues, will close this PR and redo later.
